lib/matplotlib/quiver.py

```diff
@@ -445,22 +445,20 @@ def _check_consistent_shapes(*arrays):
 
 class Quiver(mcollections.PolyCollection):
     """
-    Specialized PolyCollection for arrows.
-
-    The only API method is set_UVC(), which can be used
-    to change the size, orientation, and color of the
-    arrows; their locations are fixed when the class is
-    instantiated.  Possibly this method will be useful
-    in animations.
-
-    Much of the work in this class is done in the draw()
-    method so that as much information as possible is available
-    about the plot.  In subsequent draw() calls, recalculation
-    is limited to things that might have changed, so there
-    should be no performance penalty from putting the calculations
-    in the draw() method.
+    Specialized `.PolyCollection` for arrows.
+
+    The only API method is `set_UVC`, which can be used to change the size,
+    orientation, and color of the arrows.  Locations are changed using the
+    (inherited) `.PolyCollection.set_offsets` method.  This method may be
+    useful in animations.
```

Comment:
Unfortunately I think the behavior is a little bit more complicated than than just using `set_offsets` due to the interactions of UVC and XY which may be difficult for users to discover. See https://github.com/matplotlib/matplotlib/pull/22407